server/paymentProviders/stripe/virtual-cards.js

```diff
@@ -39,21 +39,40 @@ export const assignCardToCollective = async (cardNumber, expireDate, cvv, name,
     throw new Error('Could not find a Stripe Card matching the submitted card');
   }
 
-  const cardData = {
-    id: matchingCard.id,
-    name,
-    last4: matchingCard.last4,
-    privateData: { cardNumber, expireDate, cvv },
-    data: omit(matchingCard, ['number', 'cvc', 'exp_year', 'exp_month']),
-    CollectiveId: collectiveId,
-    HostCollectiveId: host.id,
-    UserId: userId,
-    provider: 'STRIPE',
-    spendingLimitAmount: matchingCard['spending_controls']['spending_limits'][0]['amount'],
-    spendingLimitInterval: matchingCard['spending_controls']['spending_limits'][0]['interval'].toUpperCase(),
-  };
+  return createCard(matchingCard, name, collectiveId, host.id, userId);
+};
+
+export const createCardForCollective = async (host, collectiveId, userId, name, monthlyLimit) => {
+  const connectedAccount = await host.getAccountForPaymentProvider(providerName);
+
+  const stripe = getStripeClient(host.slug, connectedAccount.token);
```

Comment:
I think we had a more sophisticated for the token, when the host is the platform account (`=== 'opencollective'`)

Reply:
It had been refactored inside the `getStripeClient`

Reply:
Cool, didn't see that.